get_root_url.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+function siteURL()
```

Comment:
Hi, because this code is tied to your specific local development environment I'm not getting any of the CSS when I run it locally. It is best for prototypes such as this to be entirely portable so that they can be picked up by anyone. Please refactor this to use local versions of the dependent CSS etc.